views/building-hours/index.js

```diff
@@ -54,6 +75,13 @@ export default class BuildingHoursView extends React.Component {
         renderRow={this._renderRow.bind(this)}
         pageSize={4}
         initialListSize={6}
+        removeClippedSubviews={false}  // remove after https://github.com/facebook/react-native/issues/8607#issuecomment-241715202
```

Comment:
Can you make a list of all the listviews that you've seen this on? Are there any you haven't?

I should add this prop to all the affected ones now, as this PR.